haas/plugins/result_handler.py

```diff
@@ -217,3 +221,63 @@ def __call__(self, result):
             self.stream.write(" '{0}'".format(result.message))
         self.stream.writeln()
         self.stream.flush()
+
+
+class SlowTestsResultHandler(IResultHandlerPlugin):
+    separator1 = '=' * 70
```

Comment:
Just not very keen on class attributes. This is a very subjective thus can be ignored